playbooks/azure/openshift-cluster/build_node_image.yml

```diff
@@ -54,10 +54,31 @@
     vars:
       os_update_latest_reboot: True
 
-- name: install openshift
+- name: install openshift GA
   import_playbook: ../../openshift-node/private/image_prep.yml
   vars:
     etcd_image: "{{ etcd_image_dict[openshift_deployment_type] }}"
+    openshift_image_tag: "v{{ openshift_azure_release }}"
+    openshift_version: "{{ openshift_azure_release }}"
+    openshift_release: "{{ openshift_azure_release }}"
+  when: (ansible_distribution == "CentOS") or (ansible_distribution == "RedHat" and openshift_azure_release != "3.10")
+
+- name: install openshift DEV
+  import_playbook: ../../openshift-node/private/image_prep.yml
+  vars:
+    etcd_image: "{{ etcd_image_dict[openshift_deployment_type] }}"
+    openshift_image_tag: "v{{ openshift_azure_release }}"
+    openshift_version: "{{ openshift_azure_release }}"
+    openshift_release: "{{ openshift_azure_release }}"
+    #this is only for dev rhel brach (non GA)
+    oreg_url: "registry.reg-aws.openshift.com/openshift3/ose-${component}:${version}"
+    oreg_auth_user: notused
+    oreg_auth_password: "{{ openshift_azure_dev_registry_token }}"
+  when: 
+  - ansible_distribution == "RedHat"
+  - openshift_azure_release == "3.10"
```

Comment:
Why do we need to hardcode the version here? Can't we just use the prerelease registry when `openshift_azure_dev_registry_token` is defined?

Reply:
It will be passed from jenkins, where it will be read from secret and passed as -e value

Reply:
What I am saying is that you could do the following:
```yaml
- name: install openshift GA
   import_playbook: ../../openshift-node/private/image_prep.yml
   vars:
     etcd_image: "{{ etcd_image_dict[openshift_deployment_type] }}"
     openshift_image_tag: "v{{ openshift_azure_release }}"
     openshift_version: "{{ openshift_azure_release }}"
     openshift_release: "{{ openshift_azure_release }}"
   when: openshift_azure_dev_registry_token == ""

- name: install openshift DEV
  import_playbook: ../../openshift-node/private/image_prep.yml
  vars:
    etcd_image: "{{ etcd_image_dict[openshift_deployment_type] }}"
    openshift_image_tag: "v{{ openshift_azure_release }}"
    openshift_version: "{{ openshift_azure_release }}"
    openshift_release: "{{ openshift_azure_release }}"
    #this is only for dev rhel brach (non GA)
    oreg_url: "registry.reg-aws.openshift.com/openshift3/ose-${component}:${version}"
    oreg_auth_user: notused
    oreg_auth_password: "{{ openshift_azure_dev_registry_token }}"
  when: openshift_azure_dev_registry_token != ""
```

Reply:
True. I would need to set the default value to "" as otherwise ansible will complain about undefined variable but this looks much nicer.